utils/static-pages/load-all.ts

```diff
@@ -0,0 +1,21 @@
+import { IArticle } from '#models/article/type';
+
+export function loadAll<T extends IArticle>(
+  articlesFolderContext: Record<string, T>
+): T[] {
+  const articles = [] as Array<T>;
+  //@ts-ignore
+  const keys = articlesFolderContext.keys();
+  const values = keys.map(articlesFolderContext);
+
+  keys
+    // weirdly context add duplicates - this filter removes them
+    .filter((k: string) => k.indexOf('./') === 0)
+    .forEach((key: string, index: number) => {
+      const slug = key.replace('.yml', '').replace('./', '');
+      //@ts-ignore
+      articles.push({ ...values[index], slug });
+    });
+
+  return articles;
+}
```

Comment:
I think these can be made even more generic and become a loader for all markdown collections but maybe this is a topic for another PR